src/scripts/markdown-checker.js

```diff
@@ -130,6 +134,16 @@ function processFrontmatter(path, lang) {
         )
       }
     }
+
+    if (
+      frontmatter.skill !== "beginner" &&
+      frontmatter.skill !== "intermediate" &&
+      frontmatter.skill !== "advanced"
+    ) {
```

Comment:
```suggestion
    if (!["beginner", "intermediate", "advanced"].includes(frontmatter.skill)) {
```
To trim down and limit repeated code =)

Reply:
One step further, could declare those strings as constant variables like
```ts
const BEGINNER = "beginner"
...
```
then reuse those below in the console log to ensure accuracy of the terms.